lib/warb/category.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+module Warb
+  module Category
+    MARKETING = "MARKETING"
+    UTILITY = "UTILITY"
```

Comment:
Como está com aspas duplas, acho que faltou rodar o rubocop para verificar se tem novas ofensas

```suggestion
    MARKETING = 'MARKETING'
    UTILITY = 'UTILITY'
```